hack/verify-licenses.sh

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `hack/verify-licenses.sh`.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+
+kube::golang::verify_go_version
+kube::util::ensure-temp-dir
+
+
+# Creating a new repository tree 
+# Deleting vendor directory to make go-licenses fetch license URLs from go-packages source repository
+git worktree add -f "${KUBE_TEMP}"/tmp_test_licenses/kubernetes HEAD >/dev/null 2>&1 || true
+cd "${KUBE_TEMP}"/tmp_test_licenses/kubernetes && rm -rf vendor
+
+
+# Ensure that we find the binaries we build before anything else.
+export GOBIN="${KUBE_OUTPUT_BINPATH}"
+PATH="${GOBIN}:${PATH}"
+
+
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
+
+
+allowed_licenses=()
+packages_flagged=()
+packages_url_missing=()
+exit_code=0
+
+
+git remote add licenses https://github.com/kubernetes/kubernetes >/dev/null 2>&1 || true
+
+
+# Install go-licenses
+echo -e "[INFO] Installing go-licenses..."
+pushd "${KUBE_TEMP}" >/dev/null
+    git clone https://github.com/google/go-licenses.git >/dev/null 2>&1
+    cd go-licenses
+    go build -o "${GOPATH}/bin"
+popd >/dev/null
+
+
+# Fetching CNCF Approved List Of Licenses
+# Refer: https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md
+curl -s 'https://spdx.org/licenses/licenses.json' > "${KUBE_TEMP}"/licenses.json
+
+
+number_of_licenses=$(jq '.licenses | length' "${KUBE_TEMP}"/licenses.json)
+loop_index_length=$(( number_of_licenses - 1 ))
+
+
+echo -e "[INFO] Fetching current list of CNCF approved licenses..."
```

Comment:
```suggestion
echo -e '[INFO] Fetching current list of CNCF approved licenses...'
```

Double quotes not needed for static strings, single quotes only